### DISCUSSION TITLE
CKAN Newbie

### DISCUSSION THREAD

**Discussion by jeswanthev** [ckan/ckan#6862] [General]
Hi All,
I am trying to evaluate CKAN for our organization and trying to setup a sample site in my local windows desktop using docker  by referring this page https://docs.ckan.org/en/2.9/maintaining/installing/install-from-docker-compose.html  and able to run CKAN and reached till this section "Build the images",  but I am not able to progress from here as I found little confusing , not sure its because of windows environment in which i am trying to setup or my little experience in this subject.  it will be great if any one has done this before has any tips for me.

**Comment by disl-pdavid** [comment]
Hi!  I'm also evaluating CKAN for my org and have recently walked through the docker installation steps (albeit on a linux server instead of windows).  What **specific** step are you having problems with, and what is going wrong?
